lib/sequin/rabbitmq/connection_cache.ex

```diff
@@ -159,7 +159,34 @@ defmodule Sequin.RabbitMq.ConnectionCache do
 
     defp default_start(%RabbitMqSink{} = sink) do
       # https://hexdocs.pm/amqp/AMQP.Connection.html#open/2-options
-      opts = [host: sink.host, port: sink.port, name: "sequin-#{sink.connection_id}"]
+      opts = [
+        host: sink.host,
+        port: sink.port,
+        virtual_host: sink.vhost,
```

Comment:
```suggestion
        virtual_host: sink.virtual_host,
```

I would spell things out but meh

Reply:
I'm down